src/ResourceManager/ServerManagement/Commands.ServerManagement.Test/ScenarioTests/ServerManagementTests.cs

```diff
@@ -0,0 +1,176 @@
+        }
+
+
+        /// <summary>
+        /// the username to use when creating the SMT NODE and Session for that node
+        /// </summary>
+        public static string NodeUserName
+        {
+            get
+            {
+                return _nodeusername ??
+                       (_nodeusername =
+                           HttpMockServer.GetVariable("SMT_NODE_USERNAME",
+                               Environment.GetEnvironmentVariable("SMT_NODE_USERNAME") ?? "gsAdmin"));
+            }
+        }
+
+        /// <summary>
+        ///  The location to use when creating resources.
+        /// </summary>
+        public static string Location
+        {
+            get
+            {
+                return _location ??
+                       (_location =
+                           HttpMockServer.GetVariable("SMT_TEST_LOCATION",
+                               Environment.GetEnvironmentVariable("SMT_TEST_LOCATION") ?? "centralus"));
+            }
+        }
+
+        /// <summary>
+        ///  the gateway name to use when creating the gateway 
+        /// </summary>
+        public static string GatewayName(int index)
+        {
+            if (!_gatewayNames.ContainsKey(index))
+            {
+                _gatewayNames[index] = HttpMockServer.GetVariable("SMT_GATEWAY_" + index,
+                    Environment.GetEnvironmentVariable("SMT_GATEWAY_" + index) ??
+                    "test_gateway_" + new Random().Next(0, int.MaxValue));
+            }
+            return _gatewayNames[index];
+        }
+
+
+        /// <summary>
+        ///  the session id to use when creating a session
+        /// </summary>
+        public static string SessionId
+        {
+            get
+            {
+                return _sessionId ??
+                       (_sessionId = HttpMockServer.GetVariable("SMT_SESSION_ID", Guid.NewGuid().ToString()));
+            }
+        }
+
+        /// <summary>
+        ///  the password to use when connecting to the node
+        /// </summary>
+        public static string NodePassword
+        {
+            // does not store actual password; on playback we don't need the real password anyway, we can just use a dummy password
+            get { return Environment.GetEnvironmentVariable("SMT_NODE_PASSWORD") ?? "S0meP@sswerd!"; }
+        }
+
+        public static bool Recording
+        {
+            get { return HttpMockServer.Mode == HttpRecorderMode.Record; }
+        }
+
+        /// <summary>
+        ///  the name of the SMT node to create 
+        /// </summary>
+        public static string NodeName
+        {
+            get
+            {
+                return _nodename ??
+                       (_nodename =
+                           HttpMockServer.GetVariable("SMT_NODE_NAME",
+                               Environment.GetEnvironmentVariable("SMT_NODE_NAME") ?? "saddlebags"));
+            }
+        }
+
+
+        [Fact]
+        public void TestGateway()
```

Comment:
@fearthecowboy please add default constructor and add following line to it https://github.com/Azure/azure-powershell/blob/dev/src/ResourceManager/Profile/Commands.Profile.Test/TenantCmdletTests.cs#L34 
otherwise your test output will not be visible during VS execution and CI run.